doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -913,6 +923,21 @@ Note that if a custom payload is used for the notification (the field `payload`
 corresponding subscription), then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
 in the notification.
 
+## Notification metadata marks
+
+In the case of using `metadataFlags` to `true` in the subscription, the following metadata
+are automatically inserted in the notification messages:
```

Comment:
This part has been re-done so probably the comment is obsolete. NTC, review the new version.